src/sentry/identity/vsts/provider.py

```diff
@@ -19,6 +19,7 @@ class VSTSIdentityProvider(OAuth2Provider):
     oauth_authorize_url = 'https://app.vssps.visualstudio.com/oauth2/authorize'
     oauth_scopes = (
         'vso.code',
+        'vso.graph',
```

Comment:
Breaks prod. need to coordinate with Cramer (or whoever else has access to the app) and recreate it with this permission added. Graph gives access to user information and membership information.